webapp/app/js/directives/clonejobs.js

```diff
@@ -93,6 +93,112 @@ treeherder.directive('thCloneJobs', [
     });
 
     $rootScope.$on(
+        thEvents.selectNextJob, function(ev){
+
+        var jobMap = ThResultSetModel.getJobMap($rootScope.repoName);
+        var lastJobSelected = ThResultSetModel.getSelectedJob($rootScope.repoName);
+        var el, key, job, step;
+
+        step = 'nextElementSibling';
+
+        // No job is selected currently , no job is selected till now ( 2 different cases )
+        if (lastJobSelected.el == undefined || lastJobSelected.el[0] == undefined ) {
```

Comment:
You can remove the last empty space before the closing condtional paren, so it is the same as the start. ie.

`undefined)`

The same is true for any conditional statements like this.